core/primitives/src/views.rs

```diff
@@ -347,10 +347,33 @@ pub struct DebugBlockStatus {
     pub timestamp_delta: u64,
 }
 
+#[cfg_attr(feature = "deepsize_feature", derive(deepsize::DeepSizeOf))]
+#[derive(Serialize, Deserialize, Debug, PartialEq, Eq)]
+pub struct PeerInfoView {
+    pub addr: String,
+    pub account_id: Option<AccountId>,
+    pub height: BlockHeight,
+    pub tracked_shards: Vec<ShardId>,
+    pub archival: bool,
+}
+
+#[cfg_attr(feature = "deepsize_feature", derive(deepsize::DeepSizeOf))]
```

Comment:
I'm not sure if the deepsize feature here is needed. I'm not too familiar with what deepsize, but I think it's used to measure struct sizes and I don't think we need to measure struct sizes here